Misc/NEWS.d/next/Core and Builtins/2018-07-24-12-54-57.bpo-33237.O95mps.rst

```diff
@@ -0,0 +1 @@
+Improved :exc:`AttributeError` message for partially initialized module.
```

Comment:
You may explain that the change impacts error during import. For example "Improved :exc:`AttributeError` message for partially initialized module on circular import."

Reply:
Hum, it's unclear to me if you like or not my proposal of adding " on circular import".